test/src/main/java/org/apache/accumulo/test/functional/ReadWriteIT.java

```diff
@@ -154,7 +154,8 @@ public void sunnyDay() throws Exception {
         if (monitorSslKeystore != null && !monitorSslKeystore.isEmpty()) {
           log.info(
               "Using HTTPS since monitor ssl keystore configuration was observed in accumulo configuration");
-          SSLContext ctx = SSLContext.getInstance("TLSv1.2");
+          SSLContext ctx =
+              SSLContext.getInstance(Property.RPC_SSL_CLIENT_PROTOCOL.getDefaultValue());
```

Comment:
I'm not sure, but this might still fit on one line if you use `var`. Could also statically import the property name, so it doesn't take up as much space on the line.